CRM/CiviAwards/Service/AwardImport.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+use CRM_CiviAwards_Helper_CaseTypeCategory as CaseTypeCategory;
+
+/**
+ * Utility class for importing Awards using CSV importer extension.
+ */
+class CRM_CiviAwards_Service_AwardImport {
+
+  /**
+   * Create an Award and the related entities.
+   *
+   * @param array $params
+   *   Information for creating the award and related entities.
+   */
+  public function create(array $params) {
+    $tx = new CRM_Core_Transaction();
+
+    $params['case_type_category'] = $this->getCaseCategoryValueForAwards();
+    $params['definition'] = $this->createDefinition($params);
+    try {
+      $caseType = civicrm_api3('CaseType', 'create', $params);
+    }
+    catch (Exception $exception) {
+      $tx->rollback();
+
+      throw new API_Exception('Exception while saving the Case Type for Award: ' . $exception->getMessage());
+    }
+    $caseTypeId = array_shift($caseType['values'])['id'];
```

Comment:
You can simply do `$caseTypeId = $caseType['id'];`

Reply:
Fixed: https://github.com/compucorp/uk.co.compucorp.civiawards/pull/164/commits/102906d9515218ea6d056dc187d940bd5e91df5a